src/containers/WelcomePage/components/LastUsedContent.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { injectT } from '@ndla/i18n';
+import { ContentResultShape } from '../../../shapes';
+import { RESOURCE_TYPE_LEARNING_PATH } from '../../../constants';
+import {
+  getContentTypeFromResourceTypes,
+  resourceToLinkProps,
+} from '../../../util/resourceHelpers';
+import { isLearningpath } from '../../../util/routeHelpers';
+import formatDate from '../../../util/formatDate';
+import { fetchDraft } from '../../../modules/draft/draftApi';
+import { classes } from '../WelcomePage';
+
+const LastUsedContent = ({ articleId, content, locale, t }) => {
+  const { contexts } = content;
+
+  const [article, setArticle] = useState(undefined);
+
+  const fetchArticle = async (articleId, locale) => {
+    const article = await fetchDraft(articleId, locale);
+    setArticle(article);
+  };
+
+  useEffect(() => {
+    if (articleId) {
+      fetchArticle(articleId, locale);
+    }
+  }, []);
+
+  let resourceType = {};
+  if (
+    contexts &&
```

Comment:
Her kan du sikkert bruke optional chaining.

Reply:
For å være helt ærlig så har jeg tatt denne koden fra SearchContent.jsx som viser frem søkeresultatet under søk. Så vet ikke om det er greit å beholde denne koden slik at de er lik?

Reply:
Det vil alltid være et poeng å forenkle og forbedre kode sjølv om den er kopiert fra ein anna plass.